Cargo.toml

```diff
@@ -88,6 +88,7 @@ tokio = { version = "1.0", default-features = false, features = [
   "macros",
   "time",
 ] }
+tokio-test = "0.4.4"
```

Comment:
Ah, so this was added. See my comment re: this in my previous PR feedback. I'd rather avoid more dependencies where we can.

Reply:
Yep, agreed. I'll yank it and use an async main as you suggested.